src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+/* #region Card */
+
+.card {
+  box-sizing: border-box;
+  width: $card_width;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__tittle {
+    margin: 0;
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $primary-text-color;
+  }
+
+  &__code {
+    margin: 0;
+    margin-bottom: 16px;
+    color: $secondary-text-color;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-weight: 400;
+    font-size: 10px;
+    margin-bottom: 24px;
+    line-height: 14px;
+
+    &-text {
+      margin: 0;
+    }
```

Comment:
You forgot to add the main color.

Reply:
I believe I have the right color :D I already stablished it